tests/unittests/Foundation/NSURLTests.m

```diff
@@ -206,7 +206,8 @@ void testNSURLMethod(SEL selector, NSURL* input, id argument, NSURL* expected) {
     ASSERT_OBJCEQ_MSG(fileURLString, newFileURLString, "File URLs do not match!");
 }
 
-TEST(NSURL, CheckResourceIsReachable) {
+// Reference platform has bug on some machines where currentDirectoryPath returns the incorrect path
```

Comment:
It's returning neither the directory from which the test is being run nor the directory the executable is in